src/main/java/com/coveros/selenified/services/Response.java

```diff
@@ -150,29 +150,73 @@ public void assertEquals(JsonArray expectedArray) {
             success = array.equals(expectedArray) ? Success.PASS : Success.FAIL;
         }
         file.recordExpected(
-                "Expected to find a response of:" + file.formatResponse(new Response(0, expectedArray, null)));
+                "Expected to find a response of: " + file.formatResponse(new Response(0, expectedArray, null)));
         file.recordActual(FOUND + file.formatResponse(this), success);
         file.addErrors(success.getErrors());
     }
 
+    /**
+     * Verifies the actual response payload is equal to the expected
+     * response payload, and writes that out to the output file
+     *
+     * @param expectedMessage - the expected response message
+     */
+    public void assertEquals(String expectedMessage) {
+        Success success = Success.FAIL;
+        if (message != null) {
+            success = message.equals(expectedMessage) ? Success.PASS : Success.FAIL;
+        }
+        file.recordExpected(
+                "Expected to find a response of: '<i>" + expectedMessage + "</i>'");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "</i>'" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)